src/components/UpcomingEventCard.vue

```diff
@@ -150,12 +157,14 @@ export default {
 		},
 	},
 	computed: {
-		link() {
-			// Find first link in description or location
-			return [
-				...(this.linkify(this.description || "") || []),
+		mainLink() {
+			// Find last link in description or location
```

Comment:
A bit cheeky, but a better default than picking the first link. I think we should return to this again at a later date, as I can still foresee this maybe causing an issue.

Reply:
I agree. Lets revisit this later on.